src/main/java/uk/gov/companieshouse/confirmationstatementapi/contoller/OfficerController.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse.confirmationstatementapi.contoller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ActiveOfficerNotFoundException;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ServiceException;
+import uk.gov.companieshouse.confirmationstatementapi.model.ActiveOfficerDetails;
+import uk.gov.companieshouse.confirmationstatementapi.service.OfficerService;
+
+@RestController
+public class OfficerController {
+
+    @Autowired
+    private OfficerService officerService;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OfficerController.class);
+
+    @GetMapping("/confirmation-statement/company/{companyNumber}/active-officer-details")
+    public ResponseEntity<ActiveOfficerDetails> getActiveDirectorDetails(@PathVariable String companyNumber) {
+        try{
+            LOGGER.info("Calling service to retrieve the active officer details.");
+            var directorDetails = officerService.getActiveDirectorDetails(companyNumber);
+            return ResponseEntity.status(HttpStatus.OK).body(directorDetails);
+        } catch (ActiveOfficerNotFoundException | ServiceException e) {
+            LOGGER.error("Error retrieving active officer details.", e);
+            return ResponseEntity.status(e.getClass() == ServiceException.class ?
```

Comment:
May be better to split the catch up?
One for ActiveOfficerNotFoundException adn one for ServiceException as we want to return different responses for each. Just to try increase code readability